## Fisica molecular



### Aproximación de Born Oppenheimer

Es una hipotesis de caracter fundamental en la que se basan en la mayoria de los calculos de la fisica molecular. Esta hipotesis se basa en que los nucleos de los atomos son mucho mas pesados y que se mueven mucho mas lentamente que los electrones. Por lo tanto, los nucleos se pueden considerar como fijos en una posicion y los electrones se pueden considerar como moviendose en un potencial fijo.

### El ion de $\text{H}_2^+$. Ligadura por tuneleo de electron. 

El ion de $\text{H}_2^+$ es la forma mas simple de una molecula diatomica. Consiste en un nucleo de dos protones y un electron. El caso de dicho ion es el unico con una solucion analitica exacta. Sin embargo, a los fines de discutir las caracteristicas principales del problema, resulta util encontrar una solucion aproximada, en un enfoque perturbativo o variacional.

![1.png](image-1.png)

Para el problema de hallar la función de onda electronica del ion de $\text{H}_2^+$, se proponen:

- $ \hat{H} $ el hamiltoniano del sistema.
- $ \psi_e $ la funcion electronica.

El hamiltoniano del sistema incluye los terminos de atracción coulombiana del electron con los protones y la energia de repulsión coulombiana entre los protones.

$$ \hat{H} = -\frac{\hbar}{2m_e}\nabla_r^2 - \frac{ke^2}{|\vec{r} + \vec{R}/2|} - \frac{ke^2}{|\vec{r} - \vec{R}/2|} + \frac{k e^2}{R} $$

En el marco de la aproximación de Born Oppenheimer, se considera la distancia $R$ como constante. Para un solo nucleo, centrado en el origen, la solucion de la ecuación de Schrödinger es conocida y es la funcion de onda del ion de hidrogeno. Para el estado de menor energia, las funciones de ondas seran los orbitales $\phi_{1s}(\vec{r - \vec{R}/2})$ y $\phi_{1s}(\vec{r + \vec{R}/2})$.

![3.png](image-3.png)

Consideramos ahora a un electron en presencia de los dos protones. Notemos que:

- Cuando el electron se encuentre en las proximidades de $\vec{r} = \pm \vec{R}/2$, se espera que la funcion de onda sea aproximadamente $\phi_{1s}(\vec{r \pm \vec{R}/2})$. 

- Hay simetria en la funcion de onda electronica respecto a la inversion de la coordenada $\vec{r} = 0$.

Las autofunciones que satisfacen las dos condiciones anteriores son de la forma:

$$ \psi_{\pm}(\vec{r}) = c_{\pm}(\phi_{1s}(\vec{r} + \vec{R}/2) \pm \phi_{1s}(\vec{r} - \vec{R}/2)) $$

Donde $N$ es una constante de normalización.



Evaluemos la energia de los estados ligantes y antiligantes de la molecula de hidrogeno ionizada $\text{H}_2^+$ en el modelo LCAO, utilizando orbitales hidrogenoides $\phi_{1s}$.

$$ E_{\pm} = \bra{\psi_{\pm}} \hat{H} \ket{\psi_{\pm}} \quad \quad \text{con} \quad \quad \psi_{\pm}(\vec{r}) = \frac{1}{\sqrt{N}}(\phi_{1s}(\vec{r} + \vec{R}/2) \pm \phi_{1s}(\vec{r} - \vec{R}/2)) $$

Denotemos:

$$ \langle{\vec{r}}\ket{\phi_a} = \phi_{1s}(\vec{r} + \vec{R}/2) \quad\quad \langle{\vec{r}}\ket{\phi_b} = \phi_{1s}(\vec{r} - \vec{R}/2) $$

Y:


$$ \hat{H} = -\frac{\hbar}{2m_e}\nabla_r^2 - \frac{ke^2}{|\vec{r} + \vec{R}/2|} - \frac{ke^2}{|\vec{r} - \vec{R}/2|} + \frac{k e^2}{R} = \hat{T} + V_a + V_b + V_{R} $$


Luego:

$$ E_{\pm} = \frac{1}{N} \bra{\phi_a \pm \phi_b}{\hat{H}}\ket{\phi_a \pm \phi_b} $$

$$ N E_{\pm} =  \bra{\phi_a}{\hat{H}}\ket{\phi_a} \pm \bra{\phi_a}{\hat{H}}\ket{\phi_b} \pm \bra{\phi_b}{\hat{H}}\ket{\phi_a} + \bra{\phi_b}{\hat{H}}\ket{\phi_b} $$

Por simetria:

$$ \bra{\phi_a}{V_b + V_R}\ket{\phi_a} = \bra{\phi_b}{V_a + V_R}\ket{\phi_b} $$
$$ \bra{\phi_a}{V_a + V_R}\ket{\phi_b} = \bra{\phi_b}{V_b + V_R}\ket{\phi_a} $$

Luego:

$$N E_{\pm} = \bra{\phi_a}\hat{T} + V_a + V_b + V_R \ket{\phi_a} + \bra{\phi_b}\hat{T} + V_a + V_b + V_R\ket{\phi_b}$$
$$+ \bra{\phi_a}\hat{T} + V_a + V_b + V_R \ket{\phi_b} + \bra{\phi_b} \hat{T} + V_a + V_b + V_R \ket{\phi_a} $$

$$N E_{\pm} = \bra{\phi_a}\hat{T} + V_a\ket{\phi_a} + \bra{\phi_a}V_b + V_R\ket{\phi_a} $$
$$+ \bra{\phi_b}\hat{T} + V_b\ket{\phi_b} + \bra{\phi_b}(V_a + V_R)\ket{\phi_b} $$
$$\pm \bra{\phi_a}\vec{T} + V_b \ket{\phi_b} \pm \bra{\phi_a} V_a + V_R \ket{\phi_b} $$
$$\pm \bra{\phi_b}\hat{T} + V_a \ket{\phi_a} \pm \bra{\phi_b} V_b + V_R\ket{\phi_a}$$

Notamos tambien que:

$$ \bra{\phi_a} \hat{T} + V_a \ket{\phi_a} = \bra{\phi_b} \hat{T} + V_b \ket{\phi_b} = E_{1s}$$

Representan la integral de la interacción de un electron con un proton, es decir un atomo de hidrogeno. Y:

$$ \bra{\phi_{a,b}} V_r \ket{\phi_{a,b}} = V_R \langle{\phi_{a,b}}\ket{\phi_{a,b}} = V_R$$

Por otra parte:

$$ \bra{\phi_b} \hat{T} + V_a \ket{\phi_a} = \bra{\phi_b} E_{1s} \ket{\phi_a} = E_{1s} \langle{\phi_b} \ket{\phi_a}$$

$$ \bra{\phi_a} \hat{T} + V_b \ket{\phi_b} = \bra{\phi_a} E_{1s} \ket{\phi_b} = E_{1s} \langle{\phi_a} \ket{\phi_b}$$

Debido a la normalización de la función de onda del hidrogeno. Denotando como $S$ a la integral de solapamiento $\langle{\phi_a}\ket{\phi_b}$, se tiene usando las igualdades anteriores:

$$ N E_{\pm} = 2E_{1s} + 2V_R \pm 2E_{1s}S + 2V_R S + 2\bra{\phi_a}V_b\ket{\phi_a} + 2\bra{\phi_a}V_a\ket{\phi_b} $$
$$ N E_{\pm} = 2(1 \pm S)(E_{1s} + V_R) + 2\bra{\phi_a}V_b\ket{\phi_a} + 2\bra{\phi_a}V_a\ket{\phi_b} $$

Se deben ahora hallar las integrales de interacción entre los orbitales $\phi_{1s}$ y los potenciales $V_a$ y $V_b$:

$$ \bra{\phi_a}V_a\ket{\phi_b} = -ke^2 \int_{\mathbb{R}^3} \frac{\phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} - \vec{R}/2)} {|\vec{r} - \vec{R}/2|} d^3r $$	

$$ \bra{\phi_a}V_b\ket{\phi_a} = -ke^2 \int_{\mathbb{R}^3} \frac{\phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} + \vec{R}/2)} {|\vec{r} - \vec{R}/2|} d^3r $$

Y la integral de overlap:


$$ \langle{\phi_a}\ket{\phi_b} = \int_{\mathbb{R}^3} \phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} - \vec{R}/2) d^3r $$

Para resolver las integrales se proponen $\vec{r_1} = \vec{r} - \frac{\vec{R}}{2}$ , $\vec{r_2} = \vec{r} + \frac{\vec{R}}{2}$. Dada la función de onda del átomo de hidrógeno en su estado fundamental:

$$\phi_{1s}(\vec{r}) = \sqrt\frac{1}{\pi a_0 ^ 3} e^{-r/a_0} $$

Con $a_0$ el radio de Bohr. Se tendra que:


$$\bra{\phi_a}V_a\ket{\phi_b} = -ke^2 \int_{\mathbb{R}^3} \frac{\phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} - \vec{R}/2)} {|\vec{r} - \vec{R}/2|} d^3r = -\frac{ke^2}{\pi a_0^3}\int_{\mathbb{R}^3} \frac{e^{-(r_1 + r_2)/a_0}}{r_1} d^3r$$

$$\bra{\phi_a}V_b\ket{\phi_a} = -ke^2 \int_{\mathbb{R}^3} \frac{\phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} + \vec{R}/2)} {|\vec{r} - \vec{R}/2|} d^3r = -\frac{ke^2}{\pi a_0^3}\int_{\mathbb{R}^3} \frac{e^{-2r_2/a_0}}{r_1} d^3r$$

$$ \langle{\phi_a}\ket{\phi_b} = \int_{\mathbb{R}^3} \phi_{1s}(\vec{r} + \vec{R}/2)  \phi_{1s}(\vec{r} - \vec{R}/2) d^3r = -\frac{1}{\pi a_0^3} \int_{\mathbb{R}^3} e^{-(r_1 + r_2)/a_0} d^3r$$

Se considera el sistema de coordenadas elipsoidales $(u,v,\varphi)$, como:

$$u = \frac{r_1 + r_2}{R} \quad\quad v = \frac{r_1 - r_2}{R} \quad\quad d^3r = \frac{1}{8}R^3(u^2 - v^2)du dv d\varphi$$

Con $u \ge 1$, $-1 \le v \le 1$ y $0 \le \varphi \le 2\pi$. Se tiene entonces:


$$\bra{\phi_a}V_a\ket{\phi_b}  = -\frac{ke^2}{\pi a_0^3} \frac{R^3}{8} \int_{0}^{2\pi} \int_{1}^{\infty} \int_{-1}^{1}  \frac{e^{-uR/a_0}}{R(u - v)/2}(u^2-v^2) d\varphi du dv $$

$$ = -\frac{ke^2 R^2}{2 a_0^3} \int_{1}^{\infty} \int_{-1}^{1} (u - v) e^{-uR/a_0} du dv$$
$$ = -\frac{ke^2 R^2}{a_0^3} \int_{1}^{\infty} u e^{-uR/a} du $$ 

En el termino con $ve^{-uR/a_0}$ se anula al integrarlo sobre $v$ por ser par. Hallamos la integral: 

$$  \int_{1}^{\infty} u e^{-uR/a_0} du =  (\frac{u}{R/a_0} e^{-uR/a_0}) \Big|_1^{\infty} - \int_{1}^{\infty} \frac{a_0}{R/a_0} e^{-uR/a_0} du = \frac{a_0}{R}e^{-R/a_0}(\frac{a_0}{R} + 1)$$

$$ \Rightarrow \bra{\phi_a}V_a\ket{\phi_b}  = -\frac{ke^2}{a_0}(1+\frac{R}{a_0})e^{-R/a_0}$$

Hallamos ahora:

$$\bra{\phi_a}V_b\ket{\phi_a} = -\frac{ke^2}{a_o^3} \frac{R^3}{8}  \int_{0}^{2\pi} \int_{1}^{\infty} \int_{-1}^{1} \frac{e^{-(u-v)R/a_0}}{R(u + v)/2}(u^2-v^2) d\varphi du dv = -\frac{ke^2 R^2}{2a_0^3} \int_{1}^{\infty} \int_{-1}^{1} (u-v)e^{-(u-v)R/a_0} du dv$$
$$ =  -\frac{ke^2 R^2}{2a_0^3} (\int_1^{\infty}ue^{-uR/a_0}du\int_{-1}^1e^{vR/a_0}dv - \int_1^{\infty} e^{-uR/a_0}du \int_{-1}^1ve^{vR/a_0}dv) $$

En donde la primera integral se calculo arriba, y:

$$\int_{-1}^1e^{vR/a_0}dv = \frac{1}{R/a_0}(e^{R/a_0} - e^{-R/a_0})$$

$$\int_{1}^{\infty} e^{-uR/a_0}du = \frac{1}{R/a_0}e^{-R/a_0} $$

$$ \int_{-1}^1ve^{vR/a_0}dv = \frac{ve^{vR/a_0}}{R/a_0}\Big|_{-1}^1 - \int_{-1}^1\frac{1}{R/a_0}e^{vR/a_0} = \frac{e^{R/a_0} + e^{-R/a_0}}{R/a_0} - \frac{e^{R/a_0} - e^{-R/a_0}}{(R/a_0)^2}$$

$$\Rightarrow \bra{\phi_a}V_b\ket{\phi_a} = -\frac{ke^2 R^2}{2a_0^3} (\frac{a_0^3}{R^3}(1+\frac{R}{a_0})(1-e^{-2R/a_0}) - \frac{a_0^3}{R^3}(\frac{R}{a_0}(1+e^{-2R/a_0}) - (1 - e^{-2R/a_0})))$$

$$ = -\frac{ke^2}{2R} ((1+\frac{R}{a_0})(1-e^{-2R/a_0}) - (\frac{R}{a_0}(1+e^{-2R/a_0}) - (1 - e^{-2R/a_0})))$$

$$ = -\frac{ke^2}{2R} (1+\frac{R}{a_0} - (1+\frac{R}{a_0})e^{-2R/a_0} - (\frac{R}{a_0} - 1 + (\frac{R}{a_0} + 1) e^{-2{R}{a_0}}))$$

$$ \Rightarrow \bra{\phi_a}V_b\ket{\phi_a}  = -\frac{ke^2}{R} (1 - (1 + \frac{R}{a_0}) e^{-2R/a_0})$$


Es necesario tambien hallar el termino de overlap $S$:

$$ \langle{\phi_a}\ket{\phi_b} = -\frac{1}{\pi a_0^3} \int_{\mathbb{R}^3} e^{-(r_1 + r_2)/a_0} d^3r = -\frac{R^3}{8\pi a_0^3} \int_{0}^{2\pi} \int_{1}^{\infty} \int_{-1}^{1} e^{-uR/a_0} (u^2 - v^2) du dv d\varphi $$

$$ = \frac{R^3}{4 a_0^3} (\int_{-1}^1 dv \int_{1}^{\infty} u^2 e^{-uR/a_0} du - \int_{-1}^1 v^2 dv \int_{1}^{\infty} e^{-uR/a_0} du )$$

$$ = \frac{R^3}{4 a_0^3} (2 (- u^2 \frac{a_0}{R} e^{-uR/a_0}\Big|_1^{\infty} + \int_{1}^{\infty} 2u \frac{a_0}{R} e^{-uR/a_0} du ) - \frac{2}{3}  \frac{a_0}{R}e^{-R/a_0})$$

$$ = \frac{R^3}{4 a_0^3} ( 2\frac{a_0}{R} e^{-R/a_0} + 4\frac{a_0^3}{R^3}e^{-R/a_0}(1+\frac{R}{a_0}) - \frac{2}{3}\frac{a_0}{R}e^{-R/a_0}) $$

$$ = \frac{e^{-R/a_0} }{2} (\frac{R^2}{a_0^2} + 2(1+\frac{R}{a_0}) - \frac{1}{3} \frac{R^2}{a_0^2}  )$$

$$ \Rightarrow \langle{\phi_a}\ket{\phi_b} = e^{-R/a_0} (1 + \frac{R}{a_0} + \frac{1}{3}\frac{R^2}{a_0^2}) \equiv S$$

### La molecula de $\text{H}_2$. Ligadura covalente y teoria de Heitler y London.

El siguiente paso en orden de complejidad, luego del estudio del ion $\text{H}_2^+$, es el estudio de la molecula de $\text{H}_2$. En este caso, se consideran dos nucleos de hidrogeno y dos electrones.

Los electrones son particulas indistinguibles, por lo que la funcion de onda total debe ser antisimetrica respecto al intercambio de las coordenadas de los electrones. Podemos empezar construir funciones de onda antisimetricas a partir de las funciones de onda de los orbitales atomicos de los atomos de hidrogeno y los estados singlete y triplete del electron.

$$ \ket{\psi}_S = \frac{1}{\sqrt{N}}(\ket{\phi_a, \phi_b} + \ket{\phi_b, \phi_a} ) \otimes \ket{\chi^S} $$

$$ \ket{\psi}_T = \frac{1}{\sqrt{N}}(\ket{\phi_a, \phi_b} - \ket{\phi_b, \phi_a} ) \otimes \ket{\chi^T} $$

Hallemos la constante de normalización $N$. Notemos que:

$$ \langle{\psi} \ket{\psi}_{S, T} = \frac{1}{N_{\pm}} (\langle{\phi_a, \phi_b} \ket{\phi_a, \phi_b} + \langle{\phi_b, \phi_a} \ket{\phi_b, \phi_a} \pm 2 \langle{\phi_a, \phi_b} \ket{\phi_b, \phi_a} ) $$

$$ \Rightarrow N_{\pm} = \int_{\mathbb{R}^3} \int_{\mathbb{R}^3} ( \phi_{1s}^2(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) \phi_{1s}^2(\vec{r_2} + \vec{\frac{\vec{R}}{2}} ) + \phi_{1s}^2(\vec{r_1} + \vec{\frac{\vec{R}}{2}}) \phi_{1s}^2(\vec{r_2} - \vec{\frac{\vec{R}}{2}} ) \\
\pm \phi_{1s}(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_2} + \vec{\frac{\vec{R}}{2}} ) \phi_{1s}(\vec{r_1} + \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_2} - \vec{\frac{\vec{R}}{2}} ) )  d^3r_1 d^3r_2$$

Notemos que:

$$ \int_{\mathbb{R}^3} \int_{\mathbb{R}^3} \phi_{1s}^2(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) \phi_{1s}^2(\vec{r_2} + \vec{\frac{\vec{R}}{2}} ) d^3r_1d^3r_2 = \int_{\mathbb{R}^3} \phi_{1s}^2(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) d^3r_1 \int_{\mathbb{R}^3} \phi_{1s}^2(\vec{r_2} + \vec{\frac{\vec{R}}{2}}) d^3r_2 = \langle{\phi_{1s}}\ket{\phi_{1s}}\langle{\phi_{1s}}\ket{\phi_{1s}} = 1$$

Analogamente:

$$ \int_{\mathbb{R}^3} \int_{\mathbb{R}^3}  \phi_{1s}^2(\vec{r_1} + \vec{\frac{\vec{R}}{2}}) \phi_{1s}^2(\vec{r_2} - \vec{\frac{\vec{R}}{2}} ) d^3r_1d^3r_2 = 1$$

Por otra parte:

$$\int_{\mathbb{R}^3} \int_{\mathbb{R}^3} \phi_{1s}(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_2} + \vec{\frac{\vec{R}}{2}} ) \phi_{1s}(\vec{r_1} + \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_2} - \vec{\frac{\vec{R}}{2}} ) d^3r_1 d^3r_2 $$

$$ = \int_{\mathbb{R}^3} \phi_{1s}(\vec{r_1} - \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_1} + \vec{\frac{\vec{R}}{2}} ) d^3r_1 \int_{\mathbb{R}^3} \phi_{1s}(\vec{r_2} + \vec{\frac{\vec{R}}{2}}) \phi_{1s}(\vec{r_2} - \vec{\frac{\vec{R}}{2}} ) d^3r_2 = S $$

En donde $S$ es la integral de solapamiento entre los orbitales $\phi_{1s}$ de los atomos de hidrogeno, calculada anteriormente. Luego:

$$ N_{\pm} = 1 + 1 \pm 2S = 2(1 \pm S) $$



Otra metodo alternativo al de Heitler y London es el de la teoria de orbitales moleculares. En este caso, supone dos electrones en un mismo estado $\ket{\psi_+}$ del ion $\text{H}_2^+$, pero con spines opuestos.

$$ \ket{\psi^{(1)}(\vec{r_1})} \sim (\phi_{a}(\vec{r_1}) + \phi_{b}(\vec{r_1}))\ket{\uparrow} $$

$$ \ket{\psi^{(2)}(\vec{r_2})} \sim (\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_2}))\ket{\downarrow} $$

La funcion de onda total se propone como:
$$ \ket{\psi(\vec{r_1}, \vec{r_2})} = \frac{1}{\sqrt{N}}(\ket{\psi^{(1)}(\vec{r_1})} \otimes \ket{\psi^{(2)}(\vec{r_2})} - \ket{\psi^{(2)}(\vec{r_1})}  \otimes  \ket{\psi^{(1)}(\vec{r_2})}) $$

$$ = \frac{1}{\sqrt{N}} (\phi_{a}(\vec{r_1}) + \phi_{b}(\vec{r_1})) (\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_2})) \ket{\uparrow\downarrow} - (\phi_{a}(\vec{r_1}) + \phi_{b}(\vec{r_1})) (\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_2})) \ket{\downarrow\uparrow}  $$

$$ = \sqrt{\frac{2}{N}} (\phi_{a}(\vec{r_1}) + \phi_{b}(\vec{r_1})) (\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_2})) (\frac{\ket{\uparrow\downarrow} - \ket{\downarrow\uparrow}}{\sqrt{2}}) $$

$$ = \sqrt{\frac{2}{N}} ( \phi_{a}(\vec{r_1})\phi_{a}(\vec{r_2}) + \phi_{a}(\vec{r_1})\phi_{b}(\vec{r_2}) + \phi_{b}(\vec{r_1})\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_1})\phi_{b}(\vec{r_2}) )\ket{\chi^S}$$

Combinando la constante de normalización con el factor de spin, se tiene:

$$\ket{\psi(\vec{r_1}, \vec{r_2})} \equiv \frac{1}{\sqrt{N}} ( \phi_{a}(\vec{r_1})\phi_{a}(\vec{r_2}) + \phi_{a}(\vec{r_1})\phi_{b}(\vec{r_2}) + \phi_{b}(\vec{r_1})\phi_{a}(\vec{r_2}) + \phi_{b}(\vec{r_1})\phi_{b}(\vec{r_2}) )\ket{\chi^S} $$

En donde $\ket{\chi^S}$ es el estado singlete de los spines. Claramente, la funcion de onda propuesta es antisimetrica respecto al intercambio de las coordenadas de los electrones.

Notemos que:

$$ \ket{\psi} = \frac{1}{\sqrt{N}} (\ket{\phi_a \phi_a} + \ket{\phi_a \phi_b} + \ket{\phi_b \phi_a} + \ket{\phi_b \phi_b}) \ket{\chi^S} $$

$$ = \ket{\psi}_{HL} + \frac{1}{\sqrt{N}} (\ket{\phi_a \phi_a} + \ket{\phi_b \phi_b}) \ket{\chi^S} $$

En donde $\ket{\psi}_{HL}$ es la función de onda de Heitler-London propuesta anteriormente.

### Energía de ionización y afinidad electronica

La ionización de la energía debe calcularse para cada ion en la tabla periódica. Como tal, para comprender la ionización de la energía, es útil comprender las ecuaciones que se utilizan para calcular la cantidad de energía necesaria para expulsar electrones.

### Ejemplo

La energía del estado fundamental de $\text{H}_2^+$ respecto al sistema $\text{H}$ en su estado fundamental y $\text{H}^+$ a distancia infinita es $-2,65 eV$.

- Calculemos la energía del $H_2^+$ respecto al sistema $\text{H}^+ + \text{H}^+ + e^-$ a distancia infinita. Notemos que:

$$\text{H}^+ + e^- \rightarrow \text{H} \quad \quad \Delta E = -13,6 \text{eV} $$

$$\text{H} + \text{H}^+ \rightarrow \text{H}_2^+ \quad \quad \Delta E = -2,65 \text{eV} $$

$$\Rightarrow \text{H}^+ + \text{H}^+ + e^- \rightarrow \text{H}_2^+ \quad \quad \Delta E = -13,6 - 2,65 = -16,25\text{eV} $$
 
La energia del $H_2^+$ respecto al sistema $\text{H}^+ + \text{H}^+ + e^-$ a distancia infinita es $-16,25 eV$, es decir, se necesitan $16,25 eV$ para separar los nucleos y el electron.

- Calculemos la energía del $H_2^+ + e^-$ respecto al sistema a distancia infinita $\text{H} + \text{H}$. Notemos que:

$$\text{H}_2^+ \rightarrow \text{H} + \text{H}^+\quad \quad \Delta E = 2,65 \text{eV} $$

$$\text{H}^+ + e^- \rightarrow \text{H} \quad \quad \Delta E = -13,6 \text{eV} $$

$$\Rightarrow \text{H}_2^+ + e^- \rightarrow \text{H} + \text{H}^+ + e^- \rightarrow \text{H} + \text{H} \quad \quad \Delta E = 2,65 - 13,6 = -10,95\text{eV} $$

- Calculemos la energía de ionización del $\text{H}_2$ si la energía de disociación es:

$$ \text{H}_2 \rightarrow \text{H}+\text{H} \quad \quad \Delta E = 4,48 \text{eV} $$

$$ \text{H} + \text{H} \rightarrow \text{H}_2^+ + e^- \quad \quad \Delta E = 10,95 \text{eV} $$

$$ \Rightarrow \text{H}_2 \rightarrow \text{H}_2^+ + e^- \quad \quad \Delta E = 4,48 + 10,95 = 15,43 \text{eV} $$

Esto es, se necesitan $15,43 eV$ para separar un electron de la molecula de $\text{H}_2$.

### Ejemplo

Una expresión fenomenológica para la energía potencial de una molécula de haluro alcalino tiene la forma:

$$ V(R) = \alpha \exp({-aR}) - \frac{e^2}{4\pi \epsilon_0 R}$$

En donde $\alpha$ es una constante, a priori desconocida. los datos espectrales para el $\text{LiF}$ proveen valores de la sepación de equilibrio $R_0 = 1.56 \text{\r{A}}$, y de la curvatura de la energía potencial $K = d^2V/dR^2|_{R=R_0} = 248\text{J}/\text{m}^2$. Hallemos la energía para separar la molecula en iones $\text{Li}^+$ y $\text{F}^-$ a una distancia infinita.

Notemos que:

$$ \frac{dV}{dR} = -\alpha a \exp({-aR}) + \frac{e^2}{4\pi \epsilon_0 R^2} \quad \quad \frac{d^2V}{dR^2} = \alpha a^2 \exp({-aR}) - \frac{e^2}{2\pi \epsilon_0 R^3} $$

Primero hallemos las constantes $a$ y $\alpha$. Veamos para un punto critico del potencial que:

$$\frac{dV}{dR}\Big|_{R_0} = 0 \Rightarrow -\alpha a \exp{(-aR_0)} + \frac{e^2}{4\pi \epsilon_0 R_0^2} = 0 \Rightarrow \alpha =  \frac{e^2}{4\pi \epsilon_0} \frac{\exp({aR_0})}{a R_0^2}$$

De modo que:

$$ \frac{d^2V}{dR^2}\Big|_{R_0} =  \frac{a e^2}{4\pi \epsilon_0} \frac{1}{R_0^2}- \frac{e^2}{2\pi \epsilon_0 R_0^3} = \frac{e^2}{4\pi \epsilon_0}(\frac{a}{R_0^2} - \frac{2}{R_0^3}) = K $$

$$ \Rightarrow a = \frac{4\pi\epsilon_0 R_0^2}{e^2} K + \frac{2}{R_0}$$

Notemos finalmente que si queremos separar la molecula desde la distancia de equilibrio a distancia infinita, el trabajo necesario es:

$$ W = \int_{R_0}^{\infty} \vec{F} d\vec{R} = - \int_{R_0}^{\infty} \frac{d}{dR} V(R) dR = V(R_0) - V(\infty)$$

Finalmente la energía necesaria para separar la molecula sera igual al trabajo necesario, finalmente reemplazando los valores obtenidos:

$$ E = \frac{e^2}{4\pi \epsilon_0} \frac{\exp(aR_0)}{aR_0^2} \exp(-aR_0) - \frac{e^2}{4\pi \epsilon_0 R_0} = \frac{e^2}{4\pi \epsilon_0 R_0}(\frac{1}{\frac{4\pi\epsilon_0 R_0^4}{e^2} K + 2R_0} - 1) $$



### Espectros rotacionales y vibracionales.

Se plantea ahora discutir de forma semi-cuantitatia, el movimiento de los nucleos de las moleculas. Utilizando la aproximación de Born Oppenheimer, se logro describir la dinamica de los electrones en una molecula y se vio que estos generan un potencial que presenta un minimo como funcion de la distancia internuclear.

Consideremos dos nucleos de masas $M_1$ y $M_2$, separados por una distancia $R$. El hamiltoniano de estos nucleos puede escribirse como:

$$ \hat{H} = \hat{H}_{CM} + \hat{H}_{rel} \quad\quad \ket{\psi} =  \ket{\psi_{CM}, \psi_{rel}}$$

En donde $\hat{H}_{CM}$ es el hamiltoniano del centro de masa y $\hat{H}_{rel}$ es el hamiltoniano de la distancia relativa entre los nucleos. Veamos los terminos relativos.

$$ (-\frac{\nabla^2}{2\mu} + \hat{V})\ket{\psi_{rel}} = E \ket{\psi_{rel}} $$

Escribimos el termino de energia cinetica en coordenadas esfericas, y proponemos una función de onda separable.

$$ -\frac{\nabla^2}{2\mu} = -\frac{\hbar^2}{2\mu} (\frac{1}{R^2} \frac{\partial}{\partial R} R^2 \frac{\partial}{\partial R} + \frac{1}{\hbar^2}\frac{\hat{L}^2}{r^2}) \quad \quad \psi(\vec{R}) = F(R)Y_l^m(\theta, \phi)$$

En donde $\hat{L}^2$ es el operador momento angular y $Y_l^m$ son los armónicos esféricos autofunciones de $\hat{L}^2$ que satisfacen $\hat{L}^2 \ket{Y_l^m} = l(l+1)\hbar^2 \ket{Y_l^m}$. Definiendo $\chi(R) = R F(R)$, se tiene, para la parte radial:

$$ -\frac{\hbar^2}{2\mu} \frac{d^2\chi}{dR^2} + (\frac{l(l+1)\hbar^2}{2\mu R^2} + V(R)) \chi(R) = E \chi(R) $$

Definiendo el potencial efectivo: 

$$U_{l}(R) \equiv V(R) + \frac{l(l+1)\hbar^2}{2\mu R^2}$$ 

Se tendra una ecuación de Schrödinger unidimensional para la distancia relativa entre los nucleos:

$$ -\frac{\hbar^2}{2\mu} \frac{d^2\chi}{dR^2} + U_{l}(R) \chi(R) = E \chi(R) $$

El efecto de aumentar el valor del momento angular $l$ es el de aumentar levemente la energia de ligadura de la molecula $R_0$. Supogamos un $R \rightarrow R_0$ y escribamos:

$$E_{rot}^l = \frac{\hbar^2 l(l+1)}{2\mu R_0^2}$$

Entonces $E_{rot}^l$ es la energia de rotación de la molecula en el estado $l$. La energia de rotación es cuantizada y depende del momento angular $l$. Teniendo en cuenta esto, podemos escribir el espectro vibracional de la molecula como:

$$E_{vib} \chi(R) = (E - E_{rot}^l)\chi(R) = -\frac{\hbar^2}{2\mu} \frac{d^2\chi}{dR^2} + V(R) \chi(R) $$

En esta aproximación el aspecto angular del estado de la molecula, no altera la dinamica radial, y ambos grados se consideran separadamente, es decir $E = E_{rot}^l + E_{vib}$. En consistencia con esta aproximación, la parte vibracional de la energía se describe por el término cuadrático del desarrollo de $V(R)$ alrededor de $R_0$, y la energia de la molecula se aproxima como:

$$E = -D_e + \hbar \omega_0 (v + \frac{1}{2}) + \frac{\hbar^2 l(l+1)}{2\mu R_0^2}$$

En donde $D_e$ es el minimo de la energia potencia, $\omega_0$ la frecuencia en la aproximación armónica y $v = 0, 1, 2, ...$ el numero cuantico vibracional.



### Espectroscopía rotacional

La espectroscopía rotacional es una técnica que permite estudiar las transiciónes entre niveles rotaciónales de las moleculas utilizando radiación de microondas.




### Ejemplo
La separación entre líneas del espectro rotacional puro del $\text{HCL}$ es de $6,35\times 10^{11} \text{Hz}$. Determinemos la separación internuclear $R_0$ de la molecula.

Según el problema $\Delta h\nu = 6,35\times 10^{11} \text{Hz}$ en U.A. Sabemos que la separación entre lineas sera:

$$\Delta (h\nu) = 2B = \frac{\hbar^2}{\mu R_0^2} $$

Con la masa reducida de la molecula:

$$\frac{1}{\mu} = \frac{1}{m_H} + \frac{1}{m_{Cl}} = \frac{m_H + m_{Cl}}{m_H m_{Cl}}$$

Luego:

$$R_0 = \sqrt{\frac{\hbar^2}{\Delta (h\nu) \mu}} = \sqrt{\frac{\hbar^2}{\Delta (h\nu)}\frac{m_H}{1 + \frac{m_H}{m_{C}}}} \approx \sqrt{\frac{\hbar^2 m_H}{\Delta (h\nu)}}$$

In [22]:
from numpy import sqrt
m_h = 1 #uma
m_cl = 35.5 #uma
nu = 6.35e11 #Hz
h = 6.62607e-34 #J*s

def uma_to_atomic_mass(uma):
    return uma / 1836.15267

def joules_to_eh(joules):
    return joules * 2.293710448e17

def atomic_radius_of_hcl(nu):
    return sqrt(uma_to_atomic_mass(m_h)/joules_to_eh(h*nu))

E = h*nu
E = joules_to_eh(E)

def atomic_lenght_to_angstroms(atomic_lenght):
    return atomic_lenght * 0.52917720859

R = sqrt(1/E/1836)
print(atomic_lenght_to_angstroms(R))

1.2571312542639648


### Ejemplo

Calculemos los primeros 3 niveles de energía de rotación de la molecula de $\text{CO}$. Determinemos tambien la longitud de onda de los fotones absorbidos en las transiciones entre estos niveles de energía, dada la longitud del enlace de $R_0 = 1,128 \text{\r{A}}$.

La masa efectiva de la molecula sera:

$$\mu = \frac{m_C m_O}{m_C + m_O}$$

Los niveles de energía seran:

$$E_{rot}^l = \frac{\hbar^2 l(l+1)}{2\mu R_0^2} = \frac{\hbar^2}{\mu R_0^2}, \frac{3\hbar^2}{\mu R_0^2}, \frac{6\hbar^2}{\mu R_0^2},...$$

### Estiramiento de moleculas

Una molecula diatomica no es rigurosamente rigida, por lo que la distancia internuclear aumenta a medida que aumenta su momento angular. Este efecto es denominado efecto de distorsion centrifuga. Dada la energia rotacional de la molecula:

$$E_{rot}^l = B l(l+1)$$

Puede verse que al aumentar su velocidad de rotación, su momento de inercia, es decir la tendencia de la molecula a oponerse a la rotación aumenta, lo que se traduce en una disminución en la energia, por lo que es necesario corregir modificar la ecuación anterior en caso de que se tenga un momento angular alto. Una expresión empirica para la energia rotacional, considerando el efecto de la distorsión centrifuga es:

$$ E_{rot}^l = B l(l+1) - D l^2(l+1)^2$$

Hallemos el valor de $D$. Supongamos que la fuerza de ligadura obedece la ley de Hook, dada una constante de fuerza $k$, y que los desplazamientos con respecto a la posición de equilibrio son pequeños. Entonces, dada la fuerza $F$:

$$ F = k(R- R_0)$$

Dada la velocidad angular de la molecula $\omega$, su fuerza centripeta sera $F_C = \mu \omega^2 R$, en el caso de que la molecula se encuentre en equilibrio, la fuerza centripeta sera igual a la fuerza de ligadura:

$$ \mu \omega^2 R = k(R - R_0) \Rightarrow R = \frac{R_0}{(1 - \frac{\mu\omega^2}{k})}$$

Para estiramientos pequeños:

$$ R - R_0 \ll R_0 \Rightarrow \frac{R - R_0}{R_0} \ll 1 \Rightarrow \frac{1}{(1 - \frac{\mu\omega^2}{k})} - 1 \ll 1 \Rightarrow 2\frac{\mu\omega^2}{k} \ll 1$$

$$ \Rightarrow \frac{1}{(1 - \frac{\mu\omega^2}{k})} \approx 1 + \frac{\mu\omega^2}{k} \Rightarrow R \approx R_0 (1 + \frac{\mu\omega^2}{k}) \Rightarrow k(R - R_0) \approx R_0\mu\omega^2$$

Reemplazando en la expresión de la energia rotacional:

$$ E_{rot}^l =  B l(l+1) + \frac{1}{2}k(R-R_0)^2  \approx \frac{\hbar^2 l(l+1)}{2\mu R_0^2} + \frac{\hat{\omega}^4 R_0^2  \mu^2}{2k}$$

De la ecuación del momento angular, dado un momento de inercia $I = \mu R_0^2$:

$$ \hat{L} = I \hat{\omega} = \mu R_0^2 \hat{\omega} \Rightarrow \frac{\omega^4 R_0^2  \mu^2}{2k} = \frac{R_0^2  \mu^2}{2k} \frac{\hat{L}^4}{\mu^4 R_0^8}$$ 

Los autovalores del operador momento angular $\hat{L}^2$ son $l(l+1)\hbar^2$, por lo que:

$$ E_{rot}^l =  B l(l+1) + \frac{1}{2}k(R-R_0)^2  \approx \frac{\hbar^2 l(l+1)}{2\mu R_0^2} + \frac{\hbar^4 l^2 (l+1)^2}{2k\mu^2 R_0^6}$$

### Potencial de Lennard-Jones

Un par de átomos o moléculas neutros están sujetos a dos fuerzas distintas en el límite de una gran separación y de una pequeña separación: una fuerza atractiva actúa a grandes distancias (fuerzas de dispersión) y una fuerza repulsiva actuando a pequeñas distancias.El potencial de Lennard-Jones es un modelo matemático sencillo para representar este comportamiento. Se propone como:

$$ V(R) = 4\epsilon \left( \left(\frac{\sigma}{R}\right)^{12} - \left( \frac{\sigma}{R}\right) ^{6} \right)$$

Donde:
- $\epsilon$ es la profundidad del pozo de potencial.
- $\sigma$ es la distancia a la cual el potencial es nulo.
- $R$ es la distancia entre los nucleos.

Como ejemplo, evaluemos la energia de punto cero del $\text{Ar}_2$ en el potencial de Lennard-Jones. Para el Argón, $\sigma = 0.340\text{nm}$ y $\epsilon=10,5\text{meV}$.

$$\frac{dV}{dR} \Big|_{R_0} =  4\epsilon \frac{1}{R_0^2}  \left( 6\left(\frac{\sigma}{R_0}\right) ^{5} -12 \left(\frac{\sigma}{R_0}\right)^{11} \right) = 0$$

$$ \Rightarrow 6\left(\frac{\sigma}{R_0}\right) ^{5} = 12 \left(\frac{\sigma}{R_0}\right)^{11} \Rightarrow \frac{1}{2} = \left(\frac{\sigma}{R_0} \right)^6 \Rightarrow R_0 = 2^{\frac{1}{6}} \sigma$$

Por otra parte la profundidad del pozo de potencial sera:

$$V(R_0) = 4\epsilon \left( \left(\frac{\sigma}{2^{1/6}\sigma}\right)^{12} - \left( \frac{\sigma}{2^{1/6}\sigma}\right) ^{6} \right) = 2\epsilon(1 - \frac{1}{2}) = -\epsilon$$

Escribiendo entonces el potencial hasta segundo orden:

$$V(R) = -\epsilon + \frac{1}{2}k(R-R_0)^2$$

$$\Rightarrow k = \frac{d^2V}{dR^2} \Big|_{R_0} = 24\epsilon \left( 26 \frac{\sigma^{12}}{R_0^{14}} - 7\frac{\sigma^{6}}{R_0^{8}} \right) = 24\epsilon \left( 26 \frac{\sigma^{12}}{\sigma^{14} 2^{14/6}} - 7\frac{\sigma^{6}}{\sigma^{8} 2^{8/6}} \right) = \frac{\epsilon}{\sigma^2} 24(\frac{26}{2^{7/3}} - \frac{7}{2^{4/3}})$$

$$\Rightarrow k = 2^{8/3}9\frac{\epsilon}{\sigma^2} = 2^{8/3}9\frac{2^{1/3}\epsilon}{R_0^2} = 72 \frac{\epsilon}{R_0^2}$$

Luego, la frecuencia de vibración de la molecula y su energia vibracional en el punto cero seran:

$$ \omega_0 = \sqrt{\frac{k}{\mu}} = \sqrt{\frac{72 \epsilon}{R_0^2 \mu}} \quad \quad E_0 \approx h \omega_0$$


### Potencial de Morse

Para muchas moléculas ligadas covalentemente, la energía potencial se describe fenoemologicamente por el potencial de Morse:

$$ V(R) = D(e^{-2a(R-R_0)} - 2e^{-a(R-R_0)})$$

Donde $D$ y $a$ son parámetros característicos de cada molécula.

Veamos un ejemplo.

La separación internuclear de la molécula de $\text{CO}$ es de $0,113\text{nm}$, la energía de disociación es de $9,60 \text{eV}$ y la frecuencia vibracional $6,51 \times 10^{13} \text{Hz}$. Determinemos los valores de $A$ y $a$.

Notemos que la energía minima que permite el potencial de Morse es $-D$. Por otro lado, la energía de disociación de una molécula viene dada por la diferencia entre
la energía mínima del potencial y la energía de punto cero. Dada la energía $E = E_{vib} + E_{rot}$. Si la molécula se encuentra en el estado fundamental, entonces $E_0 = E_{vib, 0} = \frac{1}{2}h\nu$, entonces la energía de disociación sera:

$$E_{dis} = -D - \frac{1}{2}h\nu \Rightarrow D = -E_{dis} - \frac{1}{2}h\nu$$


In [11]:
nu = 6.51e13 #Hz
h = 6.62607e-34 #J*s
E_dis = 9.60 #eV
E0 = h*nu/2

def joules_to_eV(joules):
    return joules*6.2415e18

E0 = joules_to_eV(E0)
print(-E_dis - E0)

-9.734615784770774


Luego $D=-9.73\text{eV}$. Por otra parte derivando el potencial de Morse respecto a $R$:

$$ \frac{dV}{dR} = -2aD(e^{-2a(R-R_0)} - e^{-a(R-R_0)}) \quad \quad \frac{d^2V}{dR^2} = 2a^2D(2e^{-2a(R-R_0)} - e^{-a(R-R_0)})$$

Del modelo del oscilador armomónico, tenemos que:

$$\frac{d^2V}{dR^2}\Big|_{R_0} = 2a^2D \equiv k$$

Habiamos visto que:

$$k = \frac{\hbar^2}{\mu R_0^2}$$